src/Classical/linear_code.jl

```diff
@@ -396,6 +396,19 @@ end
 #############################
      # general functions
 #############################
+function information_set(C::AbstractLinearCode)
+    """
+    Returns the indexs of the pivot columns and an information set which 
+    is the submatrix of G given by these columns.
+    """
+    if C.G == C.G_stand
+        pivot_inds=[x for x ∈ 1:C.n]
+    else
+        nonpivot_inds = _non_pivot_cols(C.G, :nsp) 
+        pivot_inds = [x for x ∈ 1:C.n if x ∉ nonpivot_inds]
```

Comment:
Nothing wrong with this, but we've stuck with the Julia notation of using `in` during the `for` loop and then the symbol `\in` for data structures.